frontend/src/charts/Treemap.svelte

```diff
@@ -57,11 +57,11 @@
     <g
       transform={`translate(${d.x0},${d.y0})`}
       use:followingTooltip={() => tooltipText(d)}
+      on:click={() => router.navigate(urlFor(`account/${d.data.account}/`))}
     >
       <rect fill={fill(d)} width={d.x1 - d.x0} height={d.y1 - d.y0} />
       <text
-        use:setOpacity={d}
-        on:click={() => router.navigate(urlFor(`account/${d.data.account}/`))}
```

Comment:
I believe this was intentional to not make the whole chart clickable. Can you revert this change, which is unrelated to the bug fixes AFAICT?

Reply:
Ah, you're correct that this wasn't described in the original pull request comment.

With the original behavior, it was still possible to click on the center of small rectangles because the text was present but transparent. Changing the text to hidden means that it becomes impossible to click without this part of the change. But the original behavior was buggy, because text labels often overlapped (invisibly).

I think that this is probably the cleanest way to go. The original click targets were basically random near small rectangles.

Reply:
As I've said, the non-clickability of most of the area was intentional (and hence the use of opacity to make even hidden account names clickable), see https://github.com/beancount/fava/issues/490#issuecomment-294529035 for some context. I'd be fine if accounts whose name isn't shown just aren't clickable anymore.

Reply:
OK, I've changed per your request.

I understand where you're coming from, but it seems very counter to Fitts's Law to minimize the clickable area. Anyways, you get to have what you want. :)

Reply:
> but it seems very counter to Fitts's Law to minimize the clickable area. 

Sure, but I currently don't really see a simple alternative without making these chart rather unusable on mobile.